.eslint.js

```diff
@@ -0,0 +1,76 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const colors = require('colors');
+const eslint = require('eslint').CLIEngine;
+
+var filesToLint = [
+    'chrome/background.js',
+    'firefox/background.js',
+    'firefox-mobile/background.js',
+    'opera/background.js',
+    'test.js'
+];
+
+var lintConfig = new eslint({
+    fix: true,
+    useEslintrc: false,
+    env: ["browser", "es6"],
+    extends: "eslint:recommended",
+    globals: [
+        "chrome: true",
+        "content: true",
+        "Components: true",
+        "XPCOMUtils: true",
+        "CustomizableUI: true",
+        "Services: true",
+        "addMessageListener: true",
+        "removeMessageListener: true",
+        "sendAsyncMessage: true"
+    ],
+    rules: {
+        "comma-dangle": 0,
+        "curly": [2, "all"],
+        "eqeqeq": [2, "smart"],
+        "indent": [2, 2],
+        "keyword-spacing": 2,
+        "linebreak-style": [2, "unix"],
+        "new-cap": [2, {"newIsCap": true, "capIsNew": false}],
+        "no-cond-assign": 0,
+        "no-spaced-func": 2,
+        "no-use-before-define": 2,
+        "no-unused-vars": ["error", { "args": "none" }],
+        "no-useless-concat": "error",
+        "one-var": [2, "never"],
+        "prefer-template": "error",
+        "quotes": ["error", "double"],
+        "semi": [2, "always"],
+        "space-before-blocks": 2,
+        "space-before-function-paren": [2, "never"],
+        "space-infix-ops": 2,
+        "space-unary-ops": 2
+    }
+});
+
+var lintReport = lintConfig.executeOnFiles(filesToLint);
+
+function logToConsole(reporFile, saveChanges) {
+    for(let i = 0; i < reporFile.results.length; ++i) {
```

Comment:
Is it possible to lint this file as well? If not, we can add a space here? `for (let...`